ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/committing/ProcessTransactionSubmission.scala

```diff
@@ -341,7 +348,9 @@ private[kvutils] class ProcessTransactionSubmission(
       else
         reject(
           recordTime,
-          buildRejectionLogEntry(transactionEntry, RejectionReason.PartyNotKnownOnLedger)
+          buildRejectionLogEntry(
+            transactionEntry,
+            RejectionReason.PartyNotKnownOnLedger("Not all parties known"))
```

Comment:
It would be lovely to know which party is unknown. It would help troubleshooting tremendously.

Reply:
Yes that would be nice.  The idea behind this change is that all ledger providers can provide better feedback.  In this case I'd rather leave it to the kvutils maintainers to decide the best message.

Reply:
I'm fine with that, we may want to address this in a separate PR.